<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawConfirmWidget</class>
 <widget class="QWidget" name="WithdrawConfirmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(52,37,90);</string>
     </property>
     <property name="text">
      <string>提现确认</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_address">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>65</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>address</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>125</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>提现手续费</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_totalNumber">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>160</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>nu</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cancel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_actualNumber">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>95</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>共需支付</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>95</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>提现数额</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>195</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>钱包密码</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_confirm">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_feeNumber">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>125</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>65</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(52,37,90);
font: 11px &quot;Microsoft YaHei UI Light&quot;;</string>
     </property>
     <property name="text">
      <string>提至地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>195</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>459</x>
       <y>11</y>
       <width>13</width>
       <height>13</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
